train.py

```diff
@@ -27,7 +27,7 @@
 WAVENET_PARAMS = './wavenet_params.json'
 STARTED_DATESTRING = "{0:%Y-%m-%dT%H-%M-%S}".format(datetime.now())
 SAMPLE_SIZE = 100000
-
+L2_REGULARIZATION_STRENGTH=0.
```

Comment:
The reason I was thinking of splitting them up is that we're currently using the `wavenet_params.json` file to encode the dimensions of the network. The file needs to stay the same between training and generation.
That's why I'm a bit reluctant to put training parameters in there as well, but I agree that we need a place to put the parameters.
Rest assured that we'll try to find a sane solution, which may also just involve putting everything into a single file. The discussion on this is in #79 and #80.